src/main/resources/META-INF/coremods.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Skin Customization Screen Transformer": "transformers/customization_screen.js"
```

Comment:
Kinda weird to have this be named `"Skin Customization Screen Transformer"`.
Making use of underscores and lowercase letters only are ideal.

Reply:
That's not supposed to be an id iirc, its the name of the transformer

Reply:
I think you're right about that IIRC.
But still, lowercase letters and underscores should be used here, as the current way you have it here breaks JSON naming conventions.
Also looking at other mods, most importantly Forge, they do not use capitals and spaces.